app/helpers/poem_helper.rb

```diff
@@ -1,2 +1,5 @@
 module PoemHelper
+  def markdown
+    Redcarpet::Markdown.new(Redcarpet::Render::HTML, extensions = {})
```

Comment:
__[rubocop]__
* Useless assignment to variable - `extensions`. [:ref](https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)